<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimerInicio</class>
 <widget class="QWidget" name="PrimerInicio">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuraciones Iniciales</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Datos Generales</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Codigo Unidad:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="codigoUnidad">
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Linea de Trabajo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineaUnidad">
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre Unidad:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="nombreUnidad">
        <property name="maxLength">
         <number>35</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2" rowspan="3" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Conexion a la base de datos:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Nombre de la base:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nombreBase"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Usuario:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usuarioBase"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Contraseña de conexion:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="contraseniaBase">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Servidor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="servidor"/>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Puerto:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="numeroPuerto">
        <property name="text">
         <string>3306</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Cuenta de Administrador:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Cuenta:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="cuenta">
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Re-Contraseña:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Contraseña:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="password">
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="repassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="nombreUsuario">
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Apellidos:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="apelldosUsuario">
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;Bienvenido, antes de iniciar por primera vez el sistema, es necesario que especifique, los datos generales de la Unidad, estos seran utilizado de forma predeterminada cada vez que inicie el sistema. Si lo que desea es instalar un cliente mas para una unidad ya existente puede, seleccionar  en tipo de configuracion 'ya existente'. Ademas debera proporcionar las opciones de conexion, esto lo puede consultar con el encargado del departamento de sistemas de su unidad.&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../ContablesPrincipal.qrc">:/iconos/kedit.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Tipo configuracion:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="btnNuevaConf">
        <property name="text">
         <string>Nueva</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="btnExisteConf">
        <property name="text">
         <string>Existente</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="botonCancelar">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="botonAceptar">
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="boxotrasConfig">
     <property name="title">
      <string>Otras configuraciones:</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Director (a) Unidad:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="director"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Jefe activo fijo:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="jefeActivo"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>codigoUnidad</tabstop>
  <tabstop>lineaUnidad</tabstop>
  <tabstop>nombreUnidad</tabstop>
  <tabstop>cuenta</tabstop>
  <tabstop>password</tabstop>
  <tabstop>repassword</tabstop>
  <tabstop>nombreUsuario</tabstop>
  <tabstop>apelldosUsuario</tabstop>
  <tabstop>nombreBase</tabstop>
  <tabstop>usuarioBase</tabstop>
  <tabstop>contraseniaBase</tabstop>
  <tabstop>servidor</tabstop>
  <tabstop>numeroPuerto</tabstop>
 </tabstops>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
 </resources>
 <connections/>
</ui>
